Makefile

```diff
@@ -139,7 +139,7 @@ gometalinter: require-gometalinter
 	    --enable=ineffassign \
 	    --enable=interfacer \
 	    --enable=maligned \
-	    --enable=megacheck \
+	    --enable=staticcheck \
```

Comment:
This might be pedantic, but could you move this down so the list of linters remains in alphabetical order?

Reply:
sure, updated